Makefile

```diff
@@ -109,9 +104,8 @@ load-synthetic-suppression-data:
 	docker-compose run api sh -c 'tmp/bcda import-suppression-directory --directory=../shared_files/synthetic1800MedicareFiles'
 
 load-fixtures-ssas:
-	docker-compose up -d db
-	docker-compose run ssas sh -c 'tmp/ssas-service --migrate'
-	docker-compose run ssas sh -c 'tmp/ssas-service --add-fixture-data'
+	docker-compose run ssas migrate -database "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -path /go/src/github.com/CMSgov/bcda-ssas-app/db/migrations up
```

Comment:
Can we run `migrate` the same way we do in the `bcda-ssas-app` repo?

Reply:
The main difference is that that in `bcda-ssas-app` we run `migrate` from a separate `migrate` image.  I'd be willing to give that strategy another try with some pairing time with you; here it would be more complicated because we'd be getting the migration files copied from a separate repo.

Reply:
Right, I was hoping we'd use the `migrate/migrate` image to be consistent with how we do it in `bcda-ssas-app` repo _as well as_ how we do it in Jenkins.

Reply:
OK; figured it out in latest commit!